### DISCUSSION TITLE
Have to sign in to github every time I launch VSC

### DISCUSSION THREAD

**Discussion by Neppkun** [community/community#7403] [Copilot Conversations]
I have to sign in to Github with Visual Studio Code every time I open VSC, is there a way of preventing this?

**Comment by benaltair** [comment]
This is happening to me as well. Didn't happen on Windows 11, but on the ARM version for MacOS it's asking me every time. This is with synced settings, so perhaps platform specific? 

PS: I wish you could add comments to the original post in Discussions - this isn't an answer.
